integration_test/add_broker_test.go

```diff
@@ -17,9 +17,9 @@ limitations under the License.
 package integration_test
 
 import (
-	"github.com/banzaicloud/go-cruise-control/integration_test/helpers"
```

Comment:
Use a local redirect/replace in go.mod instead of replacing in all files. eg: https://github.com/adobe/koperator/blob/b1a745ed16754f41a87bd4ac2f46560aff0739a1/go.mod#L142-L144

Reply:
Good call! I implemented the local replacement